### DISCUSSION TITLE
Directories in `bin` folder are confusing

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [microsoft/WinUI-Gallery#1246] [Q&A]
What am I to choose from this?
Note that none of the EXEs seem to be working.
![image](https://user-images.githubusercontent.com/65828559/229645277-ba740d92-9e68-4849-8b59-fdedd4a03a87.png)

**Comment by ghost1372** [comment]
this is a packaged app, so exe is not working is expected!

**Reply by Jay-o-Way** [reply]
Then how can I test/run the app as "standalone"? And which folder?

**Reply by ghost1372** [reply]
After the first run by Visual Studio, the app will be installed on your system, and next time you can find and run it from the start menu. Note that this is a debug app, if you need release mode, you should export your app as a msix, then you can install msix file on your system and you can find your app in start menu.
Or just change packaged mode to unpackaged mode and you can run your exe

**Reply by Jay-o-Way** [reply]
Aha. I was "used" from a different repo that it would build the app and it would create an exe file that would run :)
Still think the directory structure is weird..

**Reply by Jay-o-Way** [reply]
So, if (for example) I am changing something related to MDL2 versus Fluent icons, and I am building on a W.10 device, I would totally need _this_ to get it working on my W.11 laptop. Right?

> if you need release mode, you should export your app as a msix, then you can install msix file on your system and you can find your app in start menu

**Reply by ghost1372** [reply]
> So, if (for example) I am changing something related to MDL2 versus Fluent icons, and I am building on a W.10 device, I would totally need _this_ to get it working on my W.11 laptop. Right?
> 
> > if you need release mode, you should export your app as a msix, then you can install msix file on your system and you can find your app in start menu

No, winui app can run like uwp (packaged) or like win32 (such as wpf/winform) (unpackaged). When you change mode, many things remain unchanged (like xaml and in your situation mdl2) Only some C# code may not work properly and needs to be changed. So you need to change the mode then run the program and see which part is not working properly

**Comment by marcelwgn** [comment]
After digging around a bit, it seems publishing the app as unpackaged seems to crash with the error "Unable to load DLL 'Microsoft.UI.Xaml.dll' or one of its dependencies" and I'm a bit lost here. @bpulliam do you happen to know how to create an unpackaged version of the app?

**Reply by Jay-o-Way** [reply]
Is this still an issue with the latest version? /needinfo

**Comment by Jay-o-Way** [comment]
Ping
